packages/storybook/src/utils/versions.ts

```diff
@@ -1,4 +1,4 @@
 export const storybookAngularVersion = '5.1.9';
-export const storybookAddonKnobsVersion = '5.1.9';
+export const storybookAddonKnobsVersion = '5.0.3';
```

Comment:
Why can't we use the same version as storybookAngularVersion? 
As well, can we update to 5.10.1?